models/issues/issue.go

```diff
@@ -838,6 +838,31 @@ func AddDeletePRBranchComment(ctx context.Context, doer *user_model.User, repo *
 	return err
 }
 
+// AddTagPRCommitComment adds tag comment for pull request issue
```

Comment:
Maybe it's better to have the transaction with creating Tag.

Reply:
I'm already using transactions here though?

Reply:
ping @lunny